### PR TITLE
Add cpu usage statistics to /status command

### PR DESCRIPTION
**Pull request by danield9tqh**
## Summary
Adds CPU usage to the status command. Calculates the % of CPU the process is using across all cores of the operating system

## Testing Plan
Tested locally

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Comment by NullSoldier** [comment]
This number basically has nothing to do with Iron Fish right? We're just writing all of this code to expose their CPU usage from our app so we can show it in status. We can't ever report this number in telemetry because it won't be indicative of anything related to Iron Fish.

**Comment by danield9tqh** [comment]
Yea maybe percentage is the wrong measure then. I think showing the CPU usage of the process is helpful. Certainly for our own nodes it is helpful to see it in status since CPU percentage correlates with slowdown of other processes. Similar to how we show % of memory used and can correlate that with node slowness

File: ironfish/src/metrics/cpuMeter.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      return
+    }
+
+    const elapsedMs = now[0] - this._lastReading.time[0]
+    const elapsedCpuTime = this.totalCpuTime(osCpu) - this.totalCpuTime(this._lastReading.osCpu)
+    const elapsedProcessUserTime = processCpu.user - this._lastReading.processCpu.user
+    const elapsedProcessSysTime = processCpu.system - this._lastReading.processCpu.system
+
+    // process time is in nanoseconds, os time is in milliseconds
+    const percProcessCpu =
```
Comment (NullSoldier) [inline ironfish/src/metrics/cpuMeter.ts:88]: what does perc mean?

Comment (danield9tqh) [reply ironfish/src/metrics/cpuMeter.ts:88]: percentage

**Review by pasha121 — APPROVED** [review]
it is ok

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**